views/default/object/plugin_project/warning.php

```diff
@@ -4,11 +4,17 @@
 
 $time_created = $project->getLatestRelease()->time_created;
 
+// Get the latest Elgg version
+$latest_elgg = array_shift(elgg_get_config('elgg_versions'));
+
+// Check if there is a release compatible with latest Elgg
+$marked_compatible = $project->getRecentReleaseByElggVersion($latest_elgg);
```

Comment:
I'd prefer to evaluate this only if necessary. Maybe make into an anon function?